app.py

```diff
@@ -14,7 +14,7 @@
 from datalake.staging_stack import StagingStack
 from datalake.storage_stack import StorageStack
 
-ENV = os.environ.get("DEPLOY_ENV", "dev")
+ENV = os.environ["DEPLOY_ENV"]
```

Comment:
We could now import this from `parameter_manager`.

Reply:
No longer applicable.